### ISSUE TITLE
FreeBSD / Clang build error with v0.9.31

### ISSUE THREAD

**Issue by Jehops** [bbuchfink/diamond#337]
Hello,

Here is the error:
```
FAILED: CMakeFiles/diamond.dir/src/basic/basic.cpp.o 
/usr/local/libexec/ccache/c++  -DSTRICT_BAND -I/wrkdirs/usr/ports/biology/diamond/work/diamond-0.9.31/src -I/wrkdirs/usr/ports/biology/diamond/work/diamond-0.9.31/src/lib -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -std=gnu++11 -Wall -Wno-uninitialized -Wno-ignored-attributes -Wno-unused-variable -O2 -pipe -fstack-protector-strong -fno-strict-aliasing   -std=gnu++11 -MD -MT CMakeFiles/diamond.dir/src/basic/basic.cpp.o -MF CMakeFiles/diamond.dir/src/basic/basic.cpp.o.d -o CMakeFiles/diamond.dir/src/basic/basic.cpp.o -c /wrkdirs/usr/ports/biology/diamond/work/diamond-0.9.31/src/basic/basic.cpp
In file included from /wrkdirs/usr/ports/biology/diamond/work/diamond-0.9.31/src/basic/basic.cpp:19:
/wrkdirs/usr/ports/biology/diamond/work/diamond-0.9.31/src/basic/value.h:32:20: error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
        const std::string msg;
                          ^
/usr/include/c++/v1/iosfwd:210:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_string;
```
Here is the full build log:
http://pkg.awarnach.mathstat.dal.ca/data/12amd64-default/2020-03-19_22h50m42s/logs/errors/diamond-0.9.31.log

Please let me know if there is any useful information that I am missing or if there is anything I can test.

Regards,

Joseph

**Comment by bbuchfink** [comment]
Hi Joseph, this should be fixed in f2e307bd58ad4bca6be0755f2da5eca1dc38e1a9.

**Comment by Jehops** [comment]
[Looks good](https://svnweb.freebsd.org/ports/head/biology/diamond/Makefile?revision=528881&view=markup).  Thank you.
